addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -1504,8 +1715,41 @@ export class Rtc {
         }
     }
 
-    updateVideoDownload(rtcSession, { viewCountIncrement }) {
+    /**
+     * @param {import("@mail/discuss/call/common/rtc_session_model").RtcSession} rtcSession
+     * @param {Object} [param1]
+     * @param {number} [param1.viewCountIncrement=0] negative value to decrement
+     */
+    updateVideoDownload(rtcSession, { viewCountIncrement = 0 } = {}) {
         rtcSession.videoComponentCount += viewCountIncrement;
+        const downloadTimeout = this.downloadTimeouts.get(rtcSession.id);
+        if (downloadTimeout) {
+            this.downloadTimeouts.delete(rtcSession.id);
+            browser.clearTimeout(downloadTimeout);
+        }
+        if (this.state.connectionType === CONNECTION_TYPES.SERVER) {
+            if (rtcSession.videoComponentCount > 0) {
+                this.sfuClient?.updateDownload(rtcSession.id, [
+                    ["camera", true],
+                    ["screen", true],
+                ]);
```

Comment:
It's probably for good reason, but I can't find why we are using this strange array instead of an object?
```js
{
    camera: true,
    screen: true,
}
```

Reply:
There is no good reason, probably because it was easier to iterate as it is what you would get by doing an `object.entries()` of the equivalent object, but using an object definitely looks better, I will change that